theme/static/css/hydroshare_core.css

```diff
@@ -609,37 +639,37 @@ footer .headline h3{
     margin: 0;
 	padding: 0;
 }
+
 .social li {
     float: left;
     display: block;
     list-style-type: none;
     width: 28px;
     height: 28px;
-    background: #eee;
     margin: 0 5px 5px 0;
     text-align: center;
     line-height: 28px;
 }
+
 .social li:hover {
     background: none;
 }
-.social li, .social li:hover {
-    transition: background 0.1s;
-    -webkit-transition: background 0.1s;
-    -moz-transition: background 0.1s;
-}
+
 .social li a {
-    color: #3E7A8C;
+    color: #FFF;
 
 }
+
 .social li:hover a {
-    color: #fff;
+    color: #B5E655;
 }
-.social li a, .social li:hover a {
+
+.social li:hover a {
     transition: color 0.1s;
     -webkit-transition: color 0.1s;
     -moz-transition: color 0.1s;
```

Comment:
Though I know you didn't modify these lines, shouldn't the `-o-` and `-ms-transition` properties also be added for fail-safety?